vendor/riscv/riscv-isa-sim/customext/cv32a6embedded.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2023 Thales DIS France SAS
+//
+// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.0.
+//
+// Original Author: Zbigniew CHAMSKI <zbigniew.chamski@thalesgroup.com>
+
+#define DECODE_MACRO_USAGE_LOGGED 1
```

Comment:
What is the purpose off this macro?

Reply:
It can go away - it's a leftover caused by "copy-and-waste".  This macro is only used/relevant when a Spike extension includes additional instruction decoding based on macros.

Reply:
Can we make it go away as part of this PR?

Reply:
This is the plan indeed.

Reply:
Thanks!